nix/postgres.nix

```diff
@@ -44,6 +44,37 @@ in
           default = "./data/${config.name}";
           description = "The DB data directory";
         };
+
+        pgHbaConf =
+          let
+            mkStringOption = lib.mkOption { type = lib.types.str; };
```

Comment:
IMO we should inline this, as in future we can add a `description` here as well.